### PR TITLE
Add copy post/url/mention/code to tooltip menu

### PR DESCRIPTION
**Pull request by csduarte**
#### Summary
This PR adds additional copy options to post tooltip menu

simulator screen shot - iphone 7 - 2017-10-26 at 10 09 00

simulator screen shot - iphone 7 - 2017-10-26 at 10 09 07

simulator screen shot - iphone 7 - 2017-10-26 at 10 08 33

### CODE REVIEW

File: app/components/at_mention/at_mention.js [hunk 1]
```diff
@@ -86,6 +87,20 @@ class AtMention extends React.PureComponent {
         };
     }
 
+    handleLongPress = () => {
+        const action = {
+            text: 'Copy Mention',
```
Comment (enahum) [inline app/components/at_mention/at_mention.js]: If we are going to include this, then we need to add localization support for it

File: app/components/options_context/options_context.android.js [hunk 2]
```diff
@@ -20,16 +20,21 @@ export default class OptionsContext extends PureComponent {
         cancelText: 'Cancel'
     };
 
-    show = () => {
+    show = (additionalAction) => {
         const {actions, cancelText} = this.props;
-        if (actions.length) {
-            const actionsText = actions.map((a) => a.text);
+        let nextactions = [...actions];
```
Comment (hmhealey) [inline app/components/options_context/options_context.android.js]: nextactions should be in camel case like it is elsewhere

File: app/components/post_body/post_body.js [hunk 3]
```diff
@@ -125,7 +127,7 @@ class PostBody extends PureComponent {
         return attachments;
     }
 
-    render() {
+    render() { // eslint-disable-line complexity
```
Comment (hmhealey) [inline app/components/post_body/post_body.js]: We need to get rid of that warning :p

**Comment by jarredwitt** [comment]
@enahum changes made

**Pull request merged**